src/20_Components/amp-experiment.html

```diff
@@ -74,28 +74,36 @@
 <!--
 ... or [amp-analytics](https://www.ampproject.org/docs/reference/components/amp-analytics). Allocated variants are available as a URL substitution variable: `VARIANT(name-of-the-experiment)`. You can check the analytics requests in the network tab of the developer tools.
 -->
-<amp-analytics type="googleanalytics">
-  <script type="application/json">
-    {
-      "vars": {
-        "account": "UA-73836974-1"
-      },
-      "requests": {
-        "experiment": "${pageview}&xid=${experiment}&xvar=${variant}"
-      },
-      "triggers": {
-        "default pageview": {
-          "on": "visible",
-          "request": "experiment",
+<amp-analytics type="googleanalytics" id="analytics">
+      <script type="application/json">
+        {
           "vars": {
-            "experiment": "W4kYemYmQBSTIYNpoezCmg",
-            "variant": "VARIANT(button-color-experiment)"
+            "account": "UA-73836974-1"
+          },
+          "requests": {
+            "experiment": "${pageview}&xid=${exp}&xvar=${variant}"
+          },
+          "triggers": {
+            "default pageview": {
+              "on": "visible",
+              "request": "experiment",
+              "vars": {
+                "exp": "W4kYemYmQBSTIYNpoezCmg",
+                "variant": "VARIANT(button-color-experiment)"
+              }
+            }
           }
         }
-      }
-    }
-  </script>
-</amp-analytics>
+      </script>
+    </amp-analytics>
+  <!--
+    ... or amp-analytics configured by Google Tag Manager. After linking Google Tag manager and Google Analytics,
```

Comment:
'After linking Google Tag manager and Google Analytics' can we link to documentation?